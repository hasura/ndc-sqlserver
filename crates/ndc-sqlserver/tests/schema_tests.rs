pub mod common;

#[tokio::test]
async fn get_schema() {
    let result = common::helpers::get_schema(common::helpers::create_router().await).await;
    insta::assert_json_snapshot!(result);
}
