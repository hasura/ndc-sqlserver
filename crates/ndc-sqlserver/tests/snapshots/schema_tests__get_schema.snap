---
source: crates/ndc-sqlserver/tests/schema_tests.rs
expression: result
---
{
  "scalar_types": {},
  "object_types": {},
  "collections": [],
  "functions": [],
  "procedures": []
}
